core/Piwik.php

```diff
@@ -889,4 +889,22 @@ public static function translate($translationId, $args = array(), $language = nu
 
         return $translator->translate($translationId, $args, $language);
     }
+
+    public static function getPeriod($paramName = 'period', $default = false)
```

Comment:
I guess there is no sense in making the paramName an parameter, as it is always `period`, and the method shouldn't be used for anything else. Also providing `null` as default makes more sense

```suggestion
    public static function getPeriod($default = null)
```